src/App.js

```diff
@@ -2,7 +2,7 @@ import './App.css'
 import Oggetti from './Components/OggettiComponents/Oggetti'
 import AllCategories from './Components/Filter/AllCategories'
 import Cart from './Components/Cart/Cart'
-import Header from './data/HeaderContent'
+import Header from './Components/HeaderContent'
```

Comment:
Moved the `HeaderContent` component to the `Components` folder since it is a React component. The `data` directory should be for raw data, like JSON files or your product js file.